backend/apps/accounts/views.py

```
@@ -600,625 +120,46 @@ def post(self, request):
-        user = User.objects.filter(email__iexact=email).first()
-        if not user:
-            return Response(
-                {"error": "invalid_otp"}, status=status.HTTP_400_BAD_REQUEST
-            )
+    def get(self, request):
+        token_value = request.query_params.get('token')
 
-        token = OTPToken.objects.filter(user=user, token=otp, is_used=False).first()
-        if not token:
+        if not token_value:
             return Response(
```

Ruff (0.15.20)

[error] Undefined name `Response`

(F821)